sdk/schemaregistry/schema-registry/samples-dev/getSchemaByVersion.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates the use of a SchemaRegistryClient to register and retrieve schema.
+ */
+
+import { DefaultAzureCredential } from "@azure/identity";
+import { SchemaRegistryClient, SchemaDescription } from "@azure/schema-registry";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+// Set these environment variables or edit the following values
+const fullyQualifiedNamespace =
+  process.env["SCHEMA_REGISTRY_ENDPOINT"] || "<fullyQualifiedNamespace>";
+const groupName = process.env["SCHEMA_REGISTRY_GROUP"] || "AzureSdkSampleGroup";
+
+// Sample Avro Schema for user with first and last names
+const schemaObject = {
+  type: "record",
+  name: "User",
+  namespace: "com.azure.schemaregistry.samples",
+  fields: [
+    {
+      name: "firstName",
+      type: "string",
+    },
+    {
+      name: "lastName",
+      type: "string",
+    },
+  ],
+};
+
+const name = `${schemaObject.namespace}-${schemaObject.name}`;
+
+// Description of the schema for registration
+const schemaDescription: SchemaDescription = {
+  name,
+  groupName,
+  format: "Avro",
+  definition: JSON.stringify(schemaObject),
+};
+
+export async function main() {
+  // Create a new client
+  const client = new SchemaRegistryClient(fullyQualifiedNamespace, new DefaultAzureCredential());
+
+  // Register a schema and get back its ID.
+  const { id, version } = await client.registerSchema(schemaDescription);
```

Comment:
out of curiosity: In Python, registerSchema returns SchemaProperties (including name/format). So, we get with properties.id/properties.version/etc. Here, it looks like this is automatically returning only id and version. Is this expected?

Reply:
the rest are hidden since the sample doesn't use them, they can be exposed by just naming them, e.g.
```ts
const { id, version, name, format } = await client.registerSchema(schemaDescription);
```